modules/rgbd/src/kinfu.cpp

```diff
@@ -60,7 +61,8 @@ Ptr<Params> Params::defaultParams()
 
     // default pose of volume cube
     p.volumePose = Affine3f().translate(Vec3f(-volSize/2.f, -volSize/2.f, 0.5f));
-    p.tsdf_trunc_dist = 0.04f; //meters;
+    /* p.volumePose  = Affine3f(); */
+    p.tsdf_trunc_dist = 5 * p.voxelSize; //meters;
```

Comment:
The pose change is an experimental thing. However, in general, it is better to have the truncation distance as a multiple to voxel size, as opposed to a constant, because with different resolutions (coarse or fine) the integration might become problematic when truncation distance goes below voxel size.

Reply:
Done. Changed to 7 * voxel_size for 512 resolution and 2* voxel_size for 128 resolution